src/Crate/PDO/PDO.php

```diff
@@ -31,9 +31,10 @@ class PDO extends BasePDO implements PDOInterface
     const VERSION = '0.3.1';
     const DRIVER_NAME = 'crate';
 
-    const DSN_REGEX = '/^(?:crate)?(?::([\w\d\.-]+:\d+))+/';
+    const DSN_REGEX = '/^(?:crate)?(?::([\w\d\.-]+:\d+))+\/?([\w]+)?$/';
 
-    const ATTR_HTTP_BASIC_AUTH    = 403;
+    const CRATE_ATTR_HTTP_BASIC_AUTH    = 1000;
```

Comment:
this name change needs to be documented.
also: only deprecate `ATTR_HTTP_BASIC_AUTH` for now